client/src/document/index.tsx

```diff
@@ -156,6 +159,9 @@ export function Document(props /* TODO: define a TS interface for this */) {
   return (
     <>
       <ArticleActionsContainer doc={doc} />
+
+      <OfflineStatusBar />
```

Comment:
During server-side rendering, we don't have a `window` and therefore no `navigator`. Hence the failure I guess.